test/views/declaration/spec/ViewSpec.scala

```diff
@@ -41,6 +42,8 @@ trait ViewSpec extends PlaySpec with GuiceOneAppPerSuite with ViewValidator {
   implicit lazy val fakeRequest: FakeRequest[AnyContentAsEmpty.type] = FakeRequest("", "")
   implicit lazy val messages: Messages = messagesApi.preferred(FakeRequest())
   implicit lazy val flash: Flash = new Flash()
+  
+  
   implicit lazy val countries: List[Country] = Countries.allCountries
```

Comment:
Probably this line is incorrect. We don't need to pass countries as implicit.

Reply:
Removed